cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -5369,6 +5369,14 @@
     </description>
   </property>
 
+  <property>
+    <name>app.version.deletion.enabled</name>
```

Comment:
I think this flag is supposed to be used by all versioned endpoint, can change it to `app.legacy.version.enabled`

Reply:
I think we don't want to have the distinction between legacy version and cdap managed version in semantics right?
So should we have : `app.version.enabled` instead?
Also, yes it will be used by other versioned endpoints (not all).

Reply:
would prefer not to introduce 'legacy', maybe 'app.version.operations.enabled' or 'app.version.apis.enabled'?

Reply:
Discussed offline: Removing this flag - we will use the LCM feature flag to control enabling/disabling.